transform/variant/maf_transform.py

```diff
@@ -33,10 +34,22 @@ def _multithreading(func, lines, max_workers=5):
     """
     Create a thread pool and create alleles
     """
-    with ThreadPoolExecutor(max_workers=max_workers) as executor:
-        futures = (executor.submit(func, line) for line in lines)
-        for future in as_completed(futures):
-            yield future.result()
+    # limit queue size == number of workers
+    # see https://stackoverflow.com/questions/48263704/threadpoolexecutor-how-to-limit-the-queue-maxsize  # noqa
+
+    def chunked_iterable(iterable, chunk_size):
+        it = iter(iterable)
+        while True:
+            chunk = tuple(islice(it, chunk_size))
+            if not chunk:
+                break
+            yield chunk
```

Comment:
Does this do the same thing? https://more-itertools.readthedocs.io/en/latest/api.html#more_itertools.chunked